README.md

```diff
@@ -22,16 +25,44 @@
 
 ## Description
 
-A simple description of the service should go here
+A Microservice which stores drawings that are created in the mapviewer on s3.
 A detailed descriptions of the endpoints can be found in the [OpenAPI Spec](openapi.yaml).
 
 ### Staging Environments
 
+This service has three endpoints:
+
+- POST/kml
+- GET/kml/{id}
+- PUT/kml/{id}
+
 | Environments | URL                                                                                                                   |
 | ------------ | --------------------------------------------------------------------------------------------------------------------- |
-| DEV          | [https://service-name.bgdi-dev.swisstopo.cloud/v4/name/](https://service-name.bgdi-dev.swisstopo.cloud/v4/name/)  |
-| INT          | [https://service-name.bgdi-int.swisstopo.cloud/v4/name/](https://service-name.bgdi-int.swisstopo.cloud/v4/name/)  |
-| PROD         | [https://service-name.bgdi-prod.swisstopo.cloud/v4/name/](https://service-name.bgdi-int.swisstopo.cloud/v4/name/) |
+| DEV          | [https://service-kml.bgdi-dev.swisstopo.cloud/v4/name/](https://service-kml.bgdi-dev.swisstopo.cloud/v4/name/)  |
+| INT          | [https://service-kml.bgdi-int.swisstopo.cloud/v4/name/](https://service-kml.bgdi-int.swisstopo.cloud/v4/name/)  |
+| PROD         | [https://service-kml.bgdi-prod.swisstopo.cloud/v4/name/](https://service-kml.bgdi-int.swisstopo.cloud/v4/name/) |
+
+### POST
+
+Payload is the kml drawn in the map.
+
+| Path | Method | Content Type | Refer | Response Type|
+|------|--------|--------------|-------|--------------|
+| | POST | application/vnd.google-earth.kml+xml | map.geo.admin.ch, .bgdi.ch | application/json |
+
+### GET
+
+| Path | Method | Response Type|
+|------|--------|--------------|
+| | GET | application/json |
```

Comment:
I guess the `Response Type` could be a `application/vnd.google-earth.kml+xml` here